collectors/charts.d.plugin/ap/metadata.yaml

```
@@ -41,6 +41,9 @@ modules:
     setup:
       prerequisites:
         list:
+          - title: "Install charts.d plugin"
+            description: |
+              If using our official native DEB/RPM packages, make sure `netdata-plugin-chartsd` is installed.
```

How do people know if they are using our official native DEB/RPM packages? "I just used kickstart.sh 🤷‍♂️". How about to make the following?
```suggestion
              If [using our official native DEB/RPM packages](https://learn.netdata.cloud/docs/maintaining/update-netdata-agents#determine-which-installation-method-you-used), make sure `netdata-plugin-chartsd` is installed.
```

---

Done! Added to the rest as well.